library/std/src/net/udp/tests.rs

```diff
@@ -183,6 +183,8 @@ fn debug() {
 #[cfg_attr(any(target_os = "netbsd", target_os = "openbsd", target_os = "vxworks"), ignore)]
```

Comment:
IMHO the best solution here is not to replicate the scaling code (as that may change) but to allow small differences between the durations. For now, I think it's best to ignore this test.
```suggestion
#[cfg_attr(any(target_os = "netbsd", target_os = "openbsd", target_os = "vxworks", target_os = "nto"), ignore)]
```

Reply:
I've removed the nto-specific test code as well and disabled the tests as suggested for tcp and udp.